dotnet/src/Connectors/Connectors.Memory.CosmosDB/CosmosMemoryStore.cs

```diff
@@ -189,7 +189,7 @@ public async Task<string> UpsertAsync(string collectionName, MemoryRecord record
 
         var response = await container.UpsertItemAsync(entity, cancellationToken: cancellationToken).ConfigureAwait(false);
 
-        if (response.StatusCode == HttpStatusCode.OK || response.StatusCode == HttpStatusCode.Created)
+        if (response.StatusCode is HttpStatusCode.OK or HttpStatusCode.Created)
```

Comment:
Are you secretly trying to turn this into Python??

Reply:
Me == C# language design? :smile: